webapp/src/pages/createAccount/CreateAccount.css

```diff
@@ -2,4 +2,8 @@
   .CreateAccount {
     padding: 60px 0;
   }
+}
+
+.alert-container{
```

Comment:
I think you can get away with defining this style once in App.css, which is available to every page anyway.

Reply:
ahhh yes, thats right. 
Sorry dont know how ive missed that :)
still finding my way around your repo structure

Reply:
I've implemented the new changes.